browser/ui/views/frame/split_view/brave_multi_contents_view.cc

```diff
@@ -129,6 +143,77 @@ void BraveMultiContentsView::ResetResizeArea() {
   delegate_->ResizeWebContents(0.5, /*done_resizing=*/true);
 }
 
+void BraveMultiContentsView::UpdateContentsBorderAndOverlay() {
+  if (!contents_container_view_for_web_panel_ ||
+      !contents_container_view_for_web_panel_->GetVisible()) {
+    MultiContentsView::UpdateContentsBorderAndOverlay();
+    return;
+  }
+
+  if (!contents_container_view_for_web_panel_->IsActive()) {
+    contents_container_view_for_web_panel_->UpdateBorderAndOverlay(
+        /*is_in_split*/ true, /*is_active*/ false,
```

Comment:
Just out of curiosity, why are we passing |is_in_split| true here? Could we have a better name for the param in the future?